functions.php

```diff
@@ -0,0 +1,30 @@
+<?php
+//function to retrieve all cheeses as an array from db (with prepare statement)
+
+function getAllCheeses(PDO $db):array {
+    $query = $db->prepare("SELECT `id`, `name`, `countryoforigin`, `winepairing`, `funfact`, `imgurl` FROM `cheese`;");
+    $query->execute();
+    $result = $query->fetchAll();
+    return $result;
+};
+
+
+//display cheese items from array if array keys exist
+
+function displayACheese(array $cheeses): string {
```

Comment:
worth adding the "/**" above function to write a description of what the function is and does? That said, you do have a comment explaining this.